pytorch_blade/src/BUILD

```diff
@@ -61,15 +68,20 @@ pybind_library(
         "@local_org_torch//:torch_python",
     ],
     copts = ["-DTORCH_BLADE_BUILD_MLIR"] +
-      if_tensorrt_enabled(["-DTORCH_BLADE_BUILD_TENSORRT"])
+      if_tensorrt_enabled(["-DTORCH_BLADE_BUILD_TENSORRT"]) +
+      if_platform_alibaba(
+        ["-DTORCH_BLADE_COMMUNITY_VERSION=false"],
```

Comment:
In general, `version` means a number? The bool value is confusing.

Reply:
Done, renaming to TORCH_BLADE_PLATFORM_ALIBABA